cli/cmd/validate.go

```diff
@@ -0,0 +1,117 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/deployment"
```

Comment:
custom packages should be at bottom

Reply:
@nick4fake Please address.

Reply:
Please order this like the snippet below. Use the `goimports` tool in the future to avoid these formatting errors.
```golang
import (
	"log"

	"github.com/spf13/cobra"

	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/deployment"
	"github.com/GoogleCloudPlatform/cloud-foundation-toolkit/cli/validation"
)
```